docs/topics/mta-web-tagging-an-application.adoc

```diff
@@ -58,4 +58,56 @@ A side drawer opens.
 +
 You can see the tags attached to the application.
 +
-The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
\ No newline at end of file
+The tags can be filtered by source and tag category. The sources are *Analysis* and *Manual*. The categories are displayed in a drop-down list, for example, *HTTP*, *MVC*, *Web*, *Observability*, *Persistence*, *Application Type*, *Data Center*.
+
+[id="creating-tags_{context}"]
+== Creating tags
+
+You can create custom tags for applications for applications that {ProductShortName} assesses or analyzes.
+
+.Procedure
+
+. In *Migration* view, click *Controls*.
+. Click the *Tags* tab.
```

Comment:
@anarnold97 In the original PR, this image comes here: docs/topics/images/mta-web-assessment-tags-01.png